features/compound.feature

```diff
@@ -86,7 +84,7 @@ Feature: Test Compound Object CModel
     And I wait for AJAX to finish
     Then I press "Next"
     And wait for the page to be loaded
-    Then I should see "MARCXML File"
+    # Then I should see "MARCXML File"
```

Comment:
do you just want to take these lines out, or is it valuable to have them there as a comment?

Reply:
removed - thanks!